### PR TITLE
adds sample calls for batch payment methods

### PR DESCRIPTION
**Pull request by tnzzz**
Examples of the following methods added:
- `createBatchPaymentHistoryRecord`
- `getBatchPayment`
- `deleteBatchPayment`
- `deleteBatchPaymentByURLParam`
- `getBatchPaymentHistory`

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import * as bodyParser from "body-parser";
 import * as crypto from 'crypto';
 import express from "express";
 import { Request, Response } from "express";
-import { RepeatingInvoices, Schedule, TokenSetParameters } from 'xero-node'
+import { BatchPaymentDelete, BatchPaymentDeleteByUrlParam, HistoryRecord, RepeatingInvoices, Schedule, TokenSetParameters } from 'xero-node'
```
Comment (RettBehrens) [inline src/app.ts]: Anything from the Accounting API, please import from xero-node below. Notice that anything from the other API sets have to be imported from their respective models

File: src/app.ts [hunk 2]
```diff
@@ -70,6 +70,7 @@ import { FeedConnections, FeedConnection, CountryCode, Statements, Statement, Cr
 import { Employee as UKPayrollEmployee, Employment } from 'xero-node/dist/gen/model/payroll-uk/models';
 import { Employment as NZPayrollEmployment, EmployeeLeaveSetup as NZEmployeeLeaveSetup, Employee as NZEmployee } from 'xero-node/dist/gen/model/payroll-nz/models';
 import { ObjectGroup } from "xero-node/dist/gen/model/files/models";
+import { create } from "domain";
```
Comment (RettBehrens) [inline src/app.ts]: If this is an unused import please remove

**Review by RettBehrens — CHANGES_REQUESTED** [review]
please see comments

**Review by tnzzz — COMMENTED** [review]
Lines 2702 - 2703 are as below:

`// since we already have an Employee model in the Accounting API scope, we've imported and renamed like so:`
` // import { Employee as AUPayrollEmployee } from 'xero-node/dist/gen/model/payroll-au/models';`

Should this import statement be moved to the top? Or should all import statements relating to commented out code be moved to where it is referenced?

**Review by RettBehrens — COMMENTED** [review]
Looks good to me. The only question I have is about the changes to package-lock.json, looks like lots of subtractions?

**Comment by tnzzz** [comment]
> Looks good to me. The only question I have is about the changes to package-lock.json, looks like lots of subtractions?

I think I only ran `npm install` when I committed my initial changes, so I didn't delete delete `node_modules`. In my latest commit I have done both steps, but we've ended up with more deletions? The app itself still runs fine though 🤔

**Review by RettBehrens — APPROVED** [review]

**Pull request merged**